attacked/non_update_functions.py

```diff
@@ -0,0 +1,21 @@
+import functions
+import inspect
+from importlib import reload
+
+
+def update(request_params: str) -> str:
+    with open("functions.py", "w") as file:
+        file.write("".join(request_params))
+    reload(functions)
+    response = 'The file "functions" has been updated'
+    return True, "string_handler", [response]
+
+
+def commands_names(request_params: str) -> list:
+    names = dir(functions)

```

Comment:
instead of this consider having a varaible of functio inside `functions.py` that tells us that info